### DISCUSSION TITLE
5/5(금) 지금까지의 사용했던 라이브러리가 어떻게 동작하는지 조사해주세요.

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#125] [General]
옛날에 저도 그렇고 라이브러리를 기술이라고 생각했습니다. 하지만 라이브러리는 개발자가 쉽도록 함축된 정보를 주는 것이기 때문에 흔히들 나는 rxswift 쓸 줄알아
난 then 쓸줄알고 snpkit 쓸줄 알아! 라고 말하는 경우가 있는데 then이 어떻게 작동하는지 모르면 원래 then 사용하면 안된다고 듣었습니다. 때문에 지금 너가 사용하고 있는 then과 snpkit이 어떻게 작동하는지 조금 많이 어렵겠지만 조사해봐~~ 나도 요번에 같이 생각해보는 계기가 될 것 같아서 시켰어 🧐

**Comment by Eunho0922** [comment]
넵
